matroska_tags.xml

```diff
@@ -32,9 +32,9 @@ targeted item taken from another work of art. All tags in this list can be used
     </tag>
     <tag name="COUNTRY" class="Nesting Information" type="UTF-8">
       <description lang="en">The name of the country
-that is meant to have other tags inside (using nested tags) to country specific information about the item.
+that is meant to have other tags inside (using nested tags) to country specific information about the item,
+in the Matroska countries form; see (#country-codes) on countries codes.
```

Comment:
"countries codes" should be "country codes", just like "language codes": you only pluralize the second word.